Formula/zinit.rb

```diff
@@ -19,7 +19,6 @@ class Zinit < Formula
   uses_from_macos "zsh"
 
   def install
-    man1.install "doc/zinit.1"
     prefix.install Dir["*"]
```

Comment:
```suggestion
    prefix.install Dir["*"]
    man1.install_symlink prefix/"doc/zinit.1"
```

Reply:
Yes, it works! I had attempted to create a symlink previously, but I had not used the correct path.